Pipfile

```diff
@@ -17,6 +17,7 @@ sentry-sdk = {extras = ["flask"],version = "*"}
 gevent = "*"
 flask-apispec = "*"
 flask-swagger-ui = "*"
+pyjwt = "*"
```

Comment:
Can we pin this to <2.0.0 (see https://github.com/SwissDataScienceCenter/renku-gateway/issues/357).